models/index.js

```diff
@@ -1 +1,14 @@
+const Hero = require('./Hero');
+const Mission = require('./Mission');
+const User = require('./User');
 
+Hero.hasMany(User, {
+  foreignKey: 'hero_id',
+  onDelete: 'CASCADE',
+});
+
+Hero.hasOne(Mission, {
+  foreignKey: 'hero_id',
+});
+
```

Comment:
I think we still need
User -> Hero and
Mission -> Hero relationships defined

Reply:
I'll make the changes and repush

Reply:
I looked into the Sequalize documentation and refactored the code to reflect a one to many relationship between Hero and Missions. Plus One to One  between User and Hero. When I write out User => Hero and Hero => User I get a dependency error. Double checking the documentation it appears Sequalize intends you to specify the relationship once in a very specific manner. I'll push my changes after I finish seeding the User table.